services/cd-service/pkg/repository/transformer_db_test.go

```diff
@@ -815,7 +815,12 @@ func TestCreateApplicationVersionDB(t *testing.T) {
 					Team: "",
 				},
 			},
-			expectedDbReleases: []int64{10000},
+			expectedDbReleases: []types.ReleaseNumbers{
+				{
```

Comment:
nitpick:
I think two functions `makeReleaseNumber(int)` and `makeReleaseNumbers(int64,int64)` would reduce the lines of code in tests quite a bit